### ISSUE TITLE
[a11y]: TreeView [default] - treeview widget not operable in iPhone VO

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#14821]
### Package

@carbon/react

### Browser

Safari & iPhone

### Package version

@carbon/react@1.38.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO

### Description
Tree view widget is not operable for the iPhone VO users

Unable to expand/collapse the node in the tree view component using VO iPhone in Safari. Note: This issue is applicable for all the variants.

### Recommended Fix
Make it operable for SR users

When the standard interaction is not possible, user is unable to accomplish A task.

### WCAG 2.1 Violation

2.1.1 Keyboard
